php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -163,6 +163,14 @@ protected function writeOneof($number, $value)
         $oneof_field->setNumber($number);
     }
 
+    protected function whichOneof($oneof_name)
+    {
```

Comment:
Sorry not familiar with the oneOf API in general. Is the `whichOneOf` naming consistent with other languages?